src/wazuh_modules/vulnerability_scanner/CMakeLists.txt

```diff
@@ -53,5 +53,18 @@ set_target_properties(vulnerability_scanner PROPERTIES
         BUILD_RPATH "$ORIGIN/../lib"
 )
 
-#add_subdirectory(testtool)
-#add_subdirectory(tests)
+if(UNIT_TEST)
+  if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+    target_link_libraries(vulnerability_scanner -fprofile-arcs)
+  else()
+    target_link_libraries(vulnerability_scanner gcov)
+  endif(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+
+  add_subdirectory(tests)
+else()
+  if(FSANITIZE)
+    set(CMAKE_CXX_FLAGS_DEBUG "-g -fsanitize=address,leak,undefined")
+  endif(FSANITIZE)
+  #add_subdirectory(testtool)
```

Comment:
Is this commentary on purpose?

Reply:
Yes, no `testtool` for now or included in the scope of this issue